01-messaging.md

```diff
@@ -143,6 +145,7 @@ The receiving node:
      - MUST fail to parse the `tlv_stream`.
    - otherwise, if `type` is odd:
      - MUST discard the next `length` bytes.
+   - the odd/even rule remains in effect for custom types.
```

Comment:
I don't think this is necessary. This is already implied by the current spec imo.

Reply:
It is implied, but I added it explicitly because the question was raised during the irc meeting. I was wondering about this myself too and it could be that the reader of the spec does the same.

Reply:
Removed this, as it is also explained in the section 'when defining custom types'